### PR TITLE
Add offboarding email to IT functionality

### PR DESCRIPTION
**Pull request by machariamarigi**
#### What does this PR do?
Adds functionality for sending an email to the IT department informing them to help wipe data for an offboarding developer.

#### Description of Task to be completed?
- Template for the email was added
- A function was created to send the email.

#### Any background context you want to provide?
- Nodemailer's sending email functionality was abstracted out of all the functions that are involved with sending email. This was to aid in better error handling.

#### What are the relevant pivotal tracker stories?
[#160437746](https://www.pivotaltracker.com/story/show/160437746)

### CODE REVIEW

File: server/modules/email/emailModule.js [hunk 1]
```diff
@@ -16,12 +16,13 @@ const redirectUrl = process.env.GMAIL_REDIRECT_URL;
 const refreshToken = process.env.GMAIL_REFRESH_TOKEN;
 const user = process.env.EMAIL_USER;
 const opsEmail = process.env.OPS_EMAIL;
+const itEmail = process.env.IT_EMAIL;
```
Comment (IAMOTZ) [inline server/modules/email/emailModule.js]: Please add this(`process.env.IT_EMAIL`) along with other emails that are set through environment variables to the `.env.sample`.

File: server/modules/email/emailModule.js [hunk 2]
```diff
@@ -41,6 +42,16 @@ const emailTransport = nodemailer.createTransport({
...
 });
 
+
+const sendMail = async (mailOptions) => {
+  try {
+    await emailTransport.sendMail(mailOptions);
+    return { status: 'success' };
+  } catch (error) {
+    return { status: 'error', message: error.message };
+  }
```
Comment (IAMOTZ) [inline server/modules/email/emailModule.js]: A helper function has been created to aid the sending of response(either error or success), please make use of it here for sending both success and error response.
You can check `server/helper/response.js` to see how to use it.

File: server/modules/email/emailModule.js (hunk 1 above)
Comment (machariamarigi) [reply server/modules/email/emailModule.js]: fixed

File: server/modules/email/emailModule.js (hunk 2 above)
Comment (machariamarigi) [reply server/modules/email/emailModule.js]: implemented this

**Pull request merged**